modules/schain.js

```diff
@@ -204,4 +204,14 @@ export function setOrtbSourceExtSchain(ortbRequest, bidderRequest, context) {
   }
 }
 
+export function setOrtbSourceSchain(ortbRequest, bidderRequest, context) {
```

Comment:
I am not sure about repeating the information - I think it creates opportunity for confusion (esp. if someone in the chain wants to modify it).

Ideally we would put it in only one place with each adapter knowing where to find it. It doesn't affect only schain so I opened https://github.com/prebid/Prebid.js/issues/13006 for the wider problem.